compiler/semtypes.nim

```diff
@@ -1042,6 +1050,23 @@ proc semBlockType(c: PContext, n: PNode, prev: PType): PType =
 proc semGenericParamInInvocation(c: PContext, n: PNode): PType =
   result = semTypeNode(c, n, nil)
 
+proc semObjectTypeForInheritedGenericInst(c: PContext, n: PNode, t: PType) =
+  var check = initIntSet()
+  var realBase = t.sons[0]
+  var pos = 0
+  var base = skipTypesOrNil(realBase, skipPtrs)
```

Comment:
I think ``realBase`` and ``base`` can be ``let`` here. No need to change it, just a friendly note for the next time: It's not necessary to copy the compiler's code style when it is a leftover from the Pascal days. :-)